### PR TITLE
Rebrand Message PopUp to Teacher Moments

### PR DESCRIPTION
**Pull request by kevinrobinson**
From conversation with Deb and Justin yesterday.  This doesn't yet tackle the domain or deployment, or splitting the game out to a standalone deploy (eg., without the challenge page).  That will be a separate piece of work, this is just the renaming.

screen shot 2016-08-16 at 7 37 51 am
screen shot 2016-08-16 at 7 37 28 am

### CODE REVIEW

File: ui/src/app.jsx [hunk 1]
```diff
@@ -48,14 +48,17 @@ export default React.createClass({
...
-    '/message_popup/exploration': 'messagePopupExploration',
-    '/message_popup/evaluations/:id': 'messagePopupEvaluation',
-    '/message_popup/scoring': 'messagePopupScoring',
-    '/message_popup/author/questions' : 'messagePopupAuthorQuestions',
-    '/message_popup/author/questions/new' : 'messagePopupAuthorQuestionsNew',
-    '/message_popup/author/questions/:id' : 'messagePopupAuthorQuestionsEdit',
-    '/message_popup/audio_flow_prototype': 'messagePopupAudioFlowPrototype',
+
+    '/message_popup/*': 'messagePopupRedirect',
+    '/teachermoments': 'messagePopup',
```
Comment (kevinrobinson) [inline ui/src/app.jsx]: I'm not sure this matters (pending on deploying this to a new domain), but I kept this `teachermoments` to match what the domain name will be.  I'm assuming that when we deploy this as a standalone experience we'll remove these prefixes in the routes altogether.

**Comment by kevinrobinson** [comment]
@david-rosales Can you review?

File: ui/src/app.jsx [hunk 2]
```diff
@@ -48,14 +48,17 @@ export default React.createClass({
...
-    '/message_popup': 'messagePopup',
-    '/message_popup/exploration': 'messagePopupExploration',
-    '/message_popup/evaluations/:id': 'messagePopupEvaluation',
-    '/message_popup/scoring': 'messagePopupScoring',
-    '/message_popup/author/questions' : 'messagePopupAuthorQuestions',
-    '/message_popup/author/questions/new' : 'messagePopupAuthorQuestionsNew',
-    '/message_popup/author/questions/:id' : 'messagePopupAuthorQuestionsEdit',
-    '/message_popup/audio_flow_prototype': 'messagePopupAudioFlowPrototype',
+
+    '/message_popup/*': 'messagePopupRedirect',
```
Comment (david-rosales) [inline ui/src/app.jsx]: Should `/message_popup` also redirect?

**Comment by david-rosales** [comment]
@kevinrobinson Looks good, approved.

File: ui/src/app.jsx (hunk 2 above)
Comment (kevinrobinson) [inline ui/src/app.jsx]: Yeah good catch!  Fixed.

**Pull request merged**